Lib/test/test_format.py

```diff
@@ -519,5 +519,27 @@ def test_with_an_underscore_and_a_comma_in_format_specifier(self):
         with self.assertRaisesRegex(ValueError, error_msg):
             '{:_,}'.format(1)
 
+    @support.cpython_only
+    def test_better_error_message(self):
+        with self.assertRaises(ValueError) as e:
```

Comment:
You should use assertRaisesRegex here (and elsewhere below), to be consistent with all of the other tests in this file.

Reply:
Done.